dp/cloud/helm/dp/examples/minikube_values_nginx.yaml

```diff
@@ -0,0 +1,248 @@
+---
```

Comment:
These example values.yaml files feel quite redundant. Is there a way to reduce the redundancy across the examples?

Reply:
So we're using those files in our CI, to test both ingress controllers. I do realize that they are redundant,and I think that I can reduce them to single file with proper section commented out.